infracost-usage-example.yml

```diff
@@ -77,6 +77,7 @@ resource_type_default_usage:
     wd_queries: 11000
     wd_custom_models: 4
     wd_collections: 301
+    data-science-experience_CAPACITY_UNIT_HOURS: 1
```

Comment:
should we name this something like `data_science_experience_capacity_unit_hours` to follow the same naming conventions?

Reply:
trying something new with a POC to integrate with Demis' work. The idea would be to adopt a new naming convention for these that would align with the names the company uses for rated usage metrics

Reply:
Ohh I see, understood